src/OmniSharp.Abstractions/Endpoints.cs

```diff
@@ -64,5 +59,9 @@ public static class V2
             public const string GetCodeActions = "/v2/getcodeactions";
             public const string RunCodeAction = "/v2/runcodeaction";
         }
+        
+        public const string GetTestStartInfo = "/getteststartinfo";
```

Comment:
So this replacing the old functionality we have for running tests, so it seems like this command replaces `/gettestcontext`.

For that reason, maybe we should put it under the V2 banner?  `/v2/gettestcontext`.

Thoughts?